packages/react/src/components/MenuButton/index.tsx

```diff
@@ -178,6 +200,8 @@ const MenuButton = forwardRef<HTMLDivElement, MenuButtonProps>(
 );
 
 MenuButton.propTypes = {
+  autoAlign: PropTypes.bool,
```

Comment:
Missing code comments, something like
 `Will auto-align the MenuButton. This prop is currently experimental and is subject to future changes.`

Reply:
So there should not be any prop called autoalign in menubutton , need to remove this from menubutton as we are removing calculation of x and y coordinates for menu to be align. So size and positions are all handled by floating UI. If we keep autoalign off there will always be issues with menu not being intact with menubutton.  So there is no choice for user for autoalign it will always use floating UI for alignment


leagacyAutoAlign is a prop in menu for the components using floating UI . For example- In menuButton menu is intacted by floating UI , so sending legacyAutoalign as false, so that menu component does not calculate its position and by default its true for the components not with floating UI yet.

Reply:
Yeah MenuButton has always had built in auto-positioning logic, it's just a bit more simple (and buggy) than what we'll be providing through this PR. I agree we can remove `autoAlign` from MenuButton. I don't think any API changes in MenuButton would be required, it'll be the same as it was before this PR.